File: DataBinding/Section/LINQ/VB.NET/LINQ/ViewerForm.vb
Imports System.Text
Imports System.Xml
Imports GrapeCity.ActiveReports

Public Class ViewerForm

    Dim _manager As System.Resources.ResourceManager

    Sub New()
        Encoding.RegisterProvider(CodePagesEncodingProvider.Instance)
        InitializeComponent()
    End Sub

    '  LINQ to Object
    Private Sub ViewerForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
        _manager = New System.Resources.ResourceManager(Me.GetType)

        ' To generate a report.
        Dim rpt As New SectionReport
        rpt = New SectionReport
        rpt.LoadLayout(XmlReader.Create("..\\..\\..\\..\\rptLINQtoObject.rpx"))
        rpt.Document.Printer.PrinterName = String.Empty

        arvMain.LoadDocument(rpt)
    End Sub

End Class

File: DataBinding/Section/LINQ/C#/LINQ/ViewerForm.cs
using GrapeCity.ActiveReports;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using System.Xml;

namespace ActiveReports.Samples.LINQ
{
	public partial class ViewerForm : Form
	{
		public ViewerForm()
		{
			InitializeComponent();
		}
	  
		// Define a structure for LINQtoObject.
		private void ViewerForm_Load(object sender, EventArgs e)
		{
			// To generate a report.
			var rpt = new SectionReport();
			rpt.LoadLayout(XmlReader.Create("..\\..\\..\\..\\rptLINQtoObject.rpx"));
			rpt.Document.Printer.PrinterName = String.Empty;
			// To run the report.
			arvMain.LoadDocument(rpt);
		}
	}
}
